CHANGELOG.md

```diff
@@ -15,23 +15,27 @@ We use the following categories for changes:
 ## [Unreleased]
 
 ### Added
-- Add ability to configure chunk interval on startup
+- Add ability to configure chunk interval on startup [#991]
+- Add `ps_trace.delete_all_traces()` function to delete all trace data [#1012]
+- Add `ps_trace.set_span_retention_period(INTERVAL)` function to set span retention period. [#1015]
+- Add `ps_trace.get_span_retention_period()` database function to get current span retention period. [#1015]
+- BREAKING: Enable and configure 30 day default retention period for span data. [#1015]
```

Comment:
This probably belongs under `CHANGED` instead, since it's not adding new behavior, but changing the existing behavior (which is why it's a breaking change). As a rule, I'd assume that anything that could break things for a user must be a change, and not an addition.

Reply:
done